### PR TITLE
remove invalid warning log

### PR DESCRIPTION
**Pull request by lxning**
## Description

Please read our [CONTRIBUTING.md](https://github.com/pytorch/serve/blob/master/CONTRIBUTING.md) prior to creating your first pull request.

Please include a summary of the feature or issue being fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #(issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Feature/Issue validation/testing

Please describe the Unit or Integration tests that you ran to verify your changes and relevant result summary. Provide instructions so it can be reproduced.
Please also list any relevant details for your test configuration.

- [ ] Test A
Logs for Test A

- [ ] Test B
Logs for Test B

## Checklist:

- [ ] Did you have fun?
- [ ] Have you added tests that prove your fix is effective or that this feature works?
- [ ] Has code been commented, particularly in hard-to-understand areas?
- [ ] Have you made corresponding changes to the documentation?

### CODE REVIEW

File: frontend/server/src/main/java/org/pytorch/serve/ModelServer.java [hunk 1]
```diff
@@ -214,11 +214,6 @@ private void initModelStore() throws InvalidSnapshotException, IOException {
                             if (marMinWorkers > 0 && marMaxWorkers >= marMinWorkers) {
                                 minWorkers = marMinWorkers;
                                 maxWorkers = marMaxWorkers;
-                            } else {
```
Comment (agunapal) [inline frontend/server/src/main/java/org/pytorch/serve/ModelServer.java:217]: If someone gives faulty values like 0, -1 etc, shouldn't we have a warning?

**Review by agunapal — COMMENTED** [review]
LGTM

File: frontend/server/src/main/java/org/pytorch/serve/ModelServer.java (hunk 1 above)
Comment (lxning) [reply frontend/server/src/main/java/org/pytorch/serve/ModelServer.java:217]: This is guarded by https://github.com/pytorch/serve/blob/master/frontend/archive/src/main/java/org/pytorch/serve/archive/model/ModelConfig.java#L221

Comment (agunapal) [reply frontend/server/src/main/java/org/pytorch/serve/ModelServer.java:217]: Thanks for checking

**Review by agunapal — APPROVED** [review]
LGTM

**Pull request merged**